playbooks/group_vars/all/global.yml

```diff
@@ -16,11 +16,11 @@ force_repo_clone: true
 contra_env_setup_dir: "{{ ansible_env.HOME }}/.contra-env-setup"
 
 # Docker KVM version
-dkvm_version: v0.7.0
+dkvm_version: v0.10.0
```

Comment:
Will this driver work with 1.12.0 of minishift?

Reply:
I will need to test with 1.12.0 but then we will need to drop the patch if we want to still support v1.12.0 

https://github.com/CentOS-PaaS-SIG/contra-env-setup/pull/53/commits/fc29ab211a321d8bbeb786adc729b68e545d9259

Reply:
Bumping v0.10.0 is not enough because since 0.10.0 docker-machine-driver-kvm does not exist.
There are more versions with "distribution suffix" after name
e.g. docker-machine-driver-kvm-{centos7,alpine3.4,alpine3.5,ubuntu14.04,ubuntu16.04}

Previously it passed for me because I already had /usr/local/bin/docker-machine-driver-kvm on testing machines